stable/mongodb-replicaset/README.md

```diff
@@ -163,14 +163,14 @@ messy-hydra-mongodb-0 "primary" : "messy-hydra-mongodb-0.messy-hydra-mongodb.def
 
 Kill the primary and watch as a new master getting elected.
 ```console
-$ kubectl delete pod $RELEASE_NAME-mongodb-0
+$ kubectl delete pod $RELEASE_NAME-mongodb-replicaset-0
 
 pod "messy-hydra-mongodb-0" deleted
 ```
 
 Delete all pods and let the statefulset controller bring it up.
 ```console
-$ kubectl delete po -l app=mongodb
+$ kubectl delete po -l app=mongodb-replicaset
```

Comment:
The app label may not be sufficient. Use `-l "app=mongodb-replicaset,release=$RELEASE_NAME"`. See https://github.com/kubernetes/charts/blob/master/stable/mongodb-replicaset/templates/mongodb-service.yaml#L23-L24.